src/GovUk.Education.ExploreEducationStatistics.Data.Processor/Functions/Processor.cs

```diff
@@ -145,11 +135,7 @@ await _validatorService.Validate(message.Release.Id, subjectData, executionConte
 
                 var ex = GetInnerException(e);
 
-                await _batchService.FailImport(
-                    message.Release.Id.ToString(),
-                    message.OrigDataFileName,
-                    new List<ValidationError> {new ValidationError(ex.Message)}
-                );
+                await _batchService.FailImport(message.Release.Id, message.OrigDataFileName,new List<ValidationError> {new ValidationError(ex.Message)});
```

Comment:
Think this is less readable with it all being on one line

Reply:
Should be fixed now